src/components/Picker.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+export default function Picker({
+  head,
+  onHeadChange,
+  newCatchphrase,
+  setCatchphrase,
+  handleClick,
+  body,
+  onBodyChange,
+  accessory,
+  onAccessoryChange,
+}) {
+  // const [head, setHead] = useState("Gelfling");
+  // const [body, setBody] = useState("Gelfling");
```

Comment:
Great job using destructing props here then creating arrays to map through and create your select/option with the value of you props ex: {head} and head pics array